### DISCUSSION TITLE
interoperability of @fluentui/react-components && @fluentui/react

### DISCUSSION THREAD

**Discussion by GordonSmith** [microsoft/fluentui#21881] [General] [answered]
Is there an expectation that consumers can use both @fluentui/react-components and @fluentui/react within a single page?

**Comment by khmakoto** [comment, marked as answer]
You should be able to use both at the same time with the plan to eventually move your application completely to `@fluentui/react-components`. Feel free to read more about how to approach upgrade [here](https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/concepts-upgrading-overview--page).
